signals/sender/sender_unsolicited_sender_domain.yml

```
@@ -0,0 +1,4 @@
+name: "Sender: Unsolicited sender domain"
+type: "query"
+source: |
+  sender.email.email not in $recipient_emails
```

```suggestion
  sender.email.domain.domain not in $recipient_domains
```
guessing we also want one for email